using-kontena/vault.md

```diff
@@ -69,24 +69,52 @@ The email is needed for Let's Encrypt to notify when certificates are about to e
 
 **Note:** This registration is needed only once per Kontena Platform.
 
-#### Create domain authorization
+### Authorize the domain
 
-To be able to request certificates for a domain you must first prove that you are in charge of that domain. For this, Kontena certificate management supports both DNS and TLS-SNI verifications.
+In order to request a Let's Encrypt certificate for any domains, you must first prove that you are in control of each domain.
+Kontena's certificate management for Let's Encrypt supports different challenge types:
 
-**TLS-SNI based verification**
+* `dns-01` (since <1.0)
+* `tls-sni-01` (since 1.4)
+* `http-01` (since 1.4.x/1.5)
+
+In order to use the automated challenge methods, HTTP/HTTPS connections to the domain(s) in question must be handled by a [Kontena Load Balancer](loadbalancer.md) service.
+
+#### HTTP-01 based verification
+
+The `http-01` challenge is the most popular choice, and allows for automated certificate renewals.
+
+```bash
+$ kontena certificate authorize --type http-01 --linked-service ingress-lb/lb api.example.com
+ [done] Waiting for http-01 challenge to be deployed into my-grid/ingress-lb/lb      
+HTTP challenge is deployed, you can now request the actual certificate
+```
+
+Kontena automatically provisions the acme-challenge token/key-authorizations provided by LE to the linked service in the form of `ACME_CHALLENGE_*` environment variable secrets.
+The linked service is typically a [Kontena Load Balancer](loadbalancer.md) service which responds to `GET /.well-known/acme-challenge` requests using the provisioned tokens.
+
+Later on when requesting the certificate, the Let's Encrypt verification servers will connect to the domain on TCP port 80, and make a HTTP request for the challenge token.
+The `--linked-service` must be configured to expose port 80.
+
+#### TLS-SNI based verification
+
+The `tls-sni-01` challenge can also be used for automated certificate renewals.
```

Comment:
Should this have a warning that `tls-sni-01` challenges are currently not working?

Reply:
sure. maybe with a link to the status blog/orum on LE side?

Reply:
Linked to https://github.com/kontena/kontena/issues/3209, which has links to the LE status/blog